src/components/OutlinedCardList.tsx

```diff
@@ -88,12 +89,16 @@ export default function OutlinedCardList(props: CardProps): JSX.Element {
     number: string,
     label: string,
     unit: string,
+    secondaryLabel?: string,
   ) => {
     const { columns, rightIcon, editPath, highlighted } = props;
     const getFormattedLabel = () => (
-      <div>
-        {editPath ? getEditButton(editPath) : null}
-        <Typography align={columns ? 'center' : 'inherit'} className={props.reverse ? classes.reverseLabel : undefined}>{label}</Typography>
+      <div className={classes.headerContainer}>
+        <div style={{ display: 'flex' }}>
+          <Typography align={columns ? 'center' : 'inherit'} noWrap={props.editPath !== undefined} className={props.reverse ? classes.reverseLabel : undefined}>{label}</Typography>
```

Comment:
noWrap was added so the label doesn't overflow to a new line when the icon button is added